client/my-sites/sharing/connections/service-examples.jsx

```diff
@@ -205,6 +206,25 @@ class SharingServiceExamples extends Component {
 		];
 	}
 
+	google_my_business() {
+		return [
+			{
+				image: {
+					src: '/calypso/images/google-my-business/phone-screenshot-cropped.png',
+					alt: this.props.translate( 'Manage Google My Business locations', { textOnly: true } ),
+				},
+				label: this.props.translate(
+					'{{strong}}Connect{{/strong}} to manage your Google My Business locations directly from WordPress.com.',
+					{
+						components: {
+							strong: <strong />,
+						},
+					}
+				),
+			},
+		];
+	}
```

Comment:
We should reduce the size of this image:

<img width="367" alt="screenshot" src="https://user-images.githubusercontent.com/594356/37910962-1f0841d6-310f-11e8-8e3e-6f142a073ad5.png">

Even better, we should showcase the new stats tab. I would use the following description:

> View stats and other useful information from your Google My Business location inside WordPress.com.

Reply:
I don't mind updating the screenshots but could we at least update the copy. After giving it more thoughts, I'd go now for:

> **Connect** to view stats and other useful information from your Google My Business account inside WordPress.com.